homeassistant/components/template/light.py

```diff
@@ -439,6 +445,8 @@ def update_color(self):
 
         try:
             render = self._color_template.async_render()
+            if render in ("None", ""):
+                return
```

Comment:
Same remark.

Reply:
This one is because `self._color` is already set to none further up in the code (line 444). That said, this block of code isn't super readable / elegant, as it relies on the 'catch all' of line 444.

I've reworked it slightly to make it more inline with the other sections (temperature, value).

Reply:
I agree, this block of code can still be rework. Look at the code before my refactor, and you will see that can be worse :D